warehouse/testhelper/setup.go

```diff
@@ -167,17 +167,17 @@ func setUpJobsDB() *JobsDBResource {
 
 	var err error
 	if jobsDB.DB, err = postgres.Connect(*pgCredentials); err != nil {
-		log.Fatalf("could not connect to jobsDb with error: %s", err.Error())
+		log.Panicf("could not connect to jobsDb with error: %s", err.Error())
```

Comment:
[log. Fatal vs log. Panic](https://quasilyte.dev/blog/post/log-fatal-vs-log-panic/)

> The log.Panic vs log.Fatal is essentially panic vs os.Exit(1). The latter will not let deferred calls to run. Most of the time, it’s not what you want. It also makes testing much harder. It’s quite simple to stop panic in test by the means of recover. It’s much harder to cope with a code that does os.Exit somewhere while you’re trying to do end2end testing without loosing coverage info.

Wanted to understand if is it because of the above reason we converted it.